tests/keras/legacy/interface_test.py

```diff
@@ -28,5 +28,17 @@ def test_dense_legacy_interface():
     assert json.dumps(old_layer.get_config()) == json.dumps(new_layer.get_config())
 
 
+@keras_test
+def test_embedding_legacy_interface():
+    old_layer = keras.layers.Embedding(input_dim=4, output_dim=2, name='d',
```

Comment:
You should also add tests to cover positional arguments (here you only use keyword arguments). Current code wouldn't work for positional arguments.